ecc/decaf/constants.go

```diff
@@ -0,0 +1,28 @@
+package decaf
```

Comment:
As it's a prime-order group, the path `group/decaf448` and `group/ristretto255` might make more sense being [consistent with the naming Mike uses in his new draft.](https://github.com/bitwiseshiftleft/draft-irtf-cfrg-ristretto255-decaf448)